Rakefile

```diff
@@ -40,6 +40,7 @@ if ENV['SINATRA_ENV'] != 'production'
     config.user = 'voxpupuli'
     config.project = 'puppet_webhook'
     config.future_release = PuppetWebhook::VERSION
+    config.exclude_labels = %w{duplicate.question.invalid.wontfix.wont-fix.modulesync.skip-changelog}
```

Comment:
I think the dots break things

Reply:
Yup, fixed. That's what I get for copy/paste from one nvim window to another, instead of yank/paste